app/controllers/application_controller.rb

```diff
@@ -2009,28 +2009,14 @@ def build_saved_audit(rec, eh)
 
   def task_supported?(typ)
     vm_ids = find_checked_items.map(&:to_i).uniq
+
     if %w(migrate publish).include?(typ) && VmOrTemplate.includes_template?(vm_ids)
       render_flash_not_applicable_to_model(typ, ui_lookup(:table => "miq_template"))
       return
     end
 
-    case typ
-    when "clone"
-      if vm_ids.present? && !VmOrTemplate.cloneable?(vm_ids)
```

Comment:
Seems you're no longer calling `cloneable?` anywhere, should probably be called from `validate_clone`